app/models/solidus_bolt/bolt_configuration.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require_dependency 'solidus_bolt'
+
+module SolidusBolt
+  class BoltConfiguration < ApplicationRecord
```

Comment:
`SolidusBolt::BoltConfiguration` Bolt is repeated. Why you didn't use only `Configuration`?

Reply:
For the future, you can use Preference, Settings, or any other words.